discovery-server/pom.xml

```diff
@@ -627,7 +627,7 @@
         <dependency>
             <groupId>com.mchange</groupId>
             <artifactId>c3p0</artifactId>
-            <version>0.9.5.2</version>
+            <version>0.9.5.3</version>
```

Comment:
@alchan-lee I think c3p0 doesn't seem to be currently in use. How about removing it?

Reply:
@kyungtaak I've done it. I'll clean all dependencies up soon.